test/e2e/specs/test.js

```diff
@@ -1,18 +1,63 @@
 // For authoring Nightwatch tests, see
 // http://nightwatchjs.org/guide#usage
 
+var devServer
+
 module.exports = {
-  'default e2e tests': function (browser) {
+  before: function (browser) {
     // automatically uses dev Server port from /config.index.js
     // default: http://localhost:8080
     // see nightwatch.conf.js
-    const devServer = browser.globals.devServerURL
+    devServer = browser.globals.devServerURL
+  },
+
+  'test title': function (browser) {
+    browser
+      .url(devServer)
+      .waitForElementVisible('body', 10000)
+      .assert.title('Retirement Party')
+  },
+
+  'test header': function (browser) {
+    browser
+      .expect.element('#app > div > nav > div > div').text.to.match(/^Slides$/)
+  },
+
+  'test menu': function (browser) {
+    browser
+      .click('#app > div.application--wrap > nav > div > button.toolbar__side-icon.btn.btn--icon > div')
```

Comment:
We can use CSS IDs and call those to make these statements look a lot cleaner, rather than making the using all of the lovely css selectors. Also, it's a lot easier for devs to use the IDs rather than figuring out which selectors need to be used.